olive/workflows/run/run.py

```diff
@@ -9,10 +9,11 @@
 from pathlib import Path
 from typing import Union
 
+import importlib_metadata
```

Comment:
it seems it is third party packages?

Reply:
oh yeah, I thought it was the same as `importlib.metadata` but it's actually a backport https://pypi.org/project/importlib-metadata/

`importlib.metadata` is only available in python 3.8 and doesn't have `packages_distributions`.

What do you think we should do? 
- add `importlib_metadata` as a dependency? 
- or use `importlib.metadata` and mimic the `packages_distributions` functionality

Reply:
python < 3.8 are already deprecated and most of our other dependencies need python >= 3.8. So I don't think we need to worry about versions < 3.8?

Reply:
Updated as per offline discussion